scraper.py

```diff
@@ -0,0 +1,255 @@
+"""
+Code by : Abhishek Singh Kushwaha
+"""
+
+from bs4 import BeautifulSoup as bs
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+import time
+from configparser import ConfigParser
+import argparse
+
+
+def quit_driver(self, driver):
+        print("quitting driver...")
+        driver.quit()
```

Comment:
fix intendation. Please setup Black formatter in the project with pre-commit.

Reply:
yeah! forgot to add it to the pre-commit, have done now and have fixed the indentation as well.